PostFreeSurfer/scripts/FreeSurfer2CaretConvertAndRegisterNonlinear.sh

```diff
@@ -414,44 +414,15 @@ for Hemisphere in L R ; do
 
     #If desired, run MSMSulc folding-based registration to FS_LR initialized with FS affine
     if [ ${RegName} = "MSMSulc" ] ; then
-        #Calculate Affine Transform and Apply
-        if [ ! -e "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc ] ; then
-            mkdir "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc
```

Comment:
The lines below put outputs into "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc, so this mkdir should stay.  However, you can (should) use `mkdir -p` instead of the if statement, for simplicity.